frontend/src/components/EditRecipe/EditRecipeForm.css

```diff
@@ -22,6 +22,21 @@
   border-color: #9b9b9b;
 }
 
+select {
```

Comment:
I know these `select`, `input`, and `label` things here are related to react-select, but can we provide custom classes on those bits (like `.create-recipe-tags-select`, etc)? As is, we're styling every `select`, `input`, and `label` on the site, which I don't think is what we want.

Reply:
I deleted the styling for " select, input and label " because they were styled already in the classes. So these were just double styling.

<img width="293" alt="Screen Shot 2019-10-29 at 10 26 05 PM" src="https://user-images.githubusercontent.com/39745045/67826627-2b02f600-fa9b-11e9-8990-9544938460d7.png">